fabric-networking-api-v1/src/client/java/net/fabricmc/fabric/api/client/networking/v1/ClientPlayNetworking.java

```diff
@@ -49,9 +57,15 @@ public final class ClientPlayNetworking {
 	 * Registers a handler to a channel.
 	 * A global receiver is registered to all connections, in the present and future.
 	 *
+	 * <p>The handler runs on the network thread. After reading the buffer there, rendering
+	 * must be performed in the render thread by calling {@link ThreadExecutor#execute(Runnable)}.
```

Comment:
```suggestion
	 * <p>The handler runs on the network thread. After reading the buffer there, access to
	 * game state must be performed in the render thread by calling {@link ThreadExecutor#execute(Runnable)}.
```

Technically it's not just rendering.